src/components/InviteMemberProject/index.js

```diff
@@ -148,23 +179,54 @@ const InvaiteMemberProject = () => {
                 <TableCell>User</TableCell>
                 <TableCell align="right">Last active</TableCell>
                 <TableCell align="right">Status</TableCell>
-                <TableCell align="right">Ations</TableCell>
+                <TableCell align="right">Actions</TableCell>
               </TableRow>
             </TableHead>
             <TableBody>
-              {rows.map((row) => (
-                <TableRow
-                  key={row.name}
-                  sx={{ "&:last-child td, &:last-child th": { border: 0 } }}
-                >
-                  <TableCell component="th" scope="row">
-                    {row.name}
-                  </TableCell>
-                  <TableCell align="right">{row.lastActive}</TableCell>
-                  <TableCell align="right">{row.status}</TableCell>
-                  <TableCell align="right">{row.actions}</TableCell>
-                </TableRow>
-              ))}
+              {users && users !== undefined ? (
```

Comment:
Instead of using ternary, we can use `&&` here. Then the `else` won't be needed, which u've done below.